fedora/jellyfin-web.spec

```diff
@@ -0,0 +1,43 @@
+%global         debug_package %{nil}
+
+Name:           jellyfin-web
+Version:        10.6.0
+Release:        1%{?dist}
+Summary:        The Free Software Media System web client
+License:        GPLv3
+URL:            https://jellyfin.media
+# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%{version}.tar.gz`
+Source0:        jellyfin-web-%{version}.tar.gz
+
+%if 0%{?centos}
+BuildRequires:  yarn
+%else
+BuildRequires   nodejs-yarn
+%endif
+BuildArch:		noarch
+
+# Disable Automatic Dependency Processing
+AutoReqProv:    no
+
+%description
+Jellyfin is a free software media system that puts you in control of managing and streaming your media.
+
+
+%prep
+%autosetup -n jellyfin-web-%{version} -b 0
+
+%build
+
+%install
+yarn install
+%{__mkdir} -p %{buildroot}%{_datadir}
+mv dist %{buildroot}%{_datadir}/jellyfin-web
```

Comment:
why is this in `%install` and not in `%build`?

Reply:
I don't know, I did not write this. It works.